data/sql/updates/pending_db_world/rev_1618745790841763500.sql

```diff
@@ -0,0 +1,17 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1618745790841763500');
+INSERT INTO `gameobject` VALUES (2135397, 3240, 1, 0, 0, 1, 1, -694.417, -5674.32, 6.26285, 1.17032, -0, -0, -0.552334, -0.833623, 900, 100, 1, '', 0);
+INSERT INTO `gameobject` VALUES (2135398, 3240, 1, 0, 0, 1, 1, -1293.67, -5135, 2.13708, 1.17032, -0, -0, -0.552334, -0.833623, 900, 100, 1, '', 0);
+INSERT INTO `gameobject` VALUES (2135389, 3240, 1, 0, 0, 1, 1, -1032.82, -5515.7, 8.19592, 2.56263, -0, -0, -0.958391, -0.285457, 900, 100, 1, '', 0);
+INSERT INTO `gameobject` VALUES (2135391, 3240, 1, 0, 0, 1, 1, -657.967, -5551.35, 3.92631, 0.913286, -0, -0, -0.440938, -0.897538, 900, 100, 1, '', 0);
+INSERT INTO `gameobject` VALUES (2135392, 3240, 1, 0, 0, 1, 1, -1025.77, -5558, 5.53497, 1.86755, -0, -0, -0.80387, -0.594805, 900, 100, 1, '', 0);
+INSERT INTO `gameobject` VALUES (2135393, 3240, 1, 0, 0, 1, 1, -1254.89, -5579.15, 7.22951, 1.7521, -0, -0, -0.768216, -0.640191, 900, 100, 1, '', 0);
+INSERT INTO `gameobject` VALUES (2135395, 3240, 1, 0, 0, 1, 1, -825.151, -5593.89, 4.17995, 5.45467, -0, -0, -0.40251, 0.915416, 900, 100, 1, '', 0);
+INSERT INTO `gameobject` VALUES (2135396, 3240, 1, 0, 0, 1, 1, -701.054, -5612.3, 28.268, 1.19389, -0, -0, -0.562117, -0.827058, 900, 100, 1, '', 0);
+INSERT INTO `gameobject` VALUES (2135378, 3240, 1, 0, 0, 1, 1, -1142.09, -5103.27, 4.41125, 4.54915, -0, -0, -0.762402, 0.647104, 900, 100, 1, '', 0);
+INSERT INTO `gameobject` VALUES (2135379, 3240, 1, 0, 0, 1, 1, -969.367, -5182.59, 1.80058, 1.82394, -0, -0, -0.790712, -0.612188, 900, 100, 1, '', 0);
+INSERT INTO `gameobject` VALUES (2135385, 3240, 1, 0, 0, 1, 1, -813.653, -5316.14, 3.31857, 4.71632, -0, -0, -0.705714, 0.708497, 900, 100, 1, '', 0);
+INSERT INTO `gameobject` VALUES (2135387, 3240, 1, 0, 0, 1, 1, -1251.37, -5341.21, 5.36343, 4.04087, -0, -0, -0.900604, 0.43464, 900, 100, 1, '', 0);
+INSERT INTO `gameobject` VALUES (2135386, 3240, 1, 0, 0, 1, 1, -1140.34, -5337.26, 5.67458, 4.04087, -0, -0, -0.900604, 0.43464, 900, 100, 1, '', 0);
+INSERT INTO `gameobject` VALUES (2135384, 3240, 1, 0, 0, 1, 1, -1518.92, -5304.11, 7.73214, 3.61284, -0, -0, -0.972369, 0.233449, 900, 100, 1, '', 0);
+INSERT INTO `gameobject` VALUES (2135388, 3240, 1, 0, 0, 1, 1, -1327.5, -5496.98, 5.54005, 2.56263, -0, -0, -0.958391, -0.285457, 900, 0, 1, '', 0);
```

Comment:
hello! thanks for your contribution @Branel 

just some minor notes:

- you should always use `DELETE` before `INSERT`
- you should combine multiple `INSERT` statements into one

for more details, please read: https://www.azerothcore.org/wiki/Dealing-with-SQL-files

Reply:
Thank you for the feedback!